docs/units-and-global-variables.rst

```diff
@@ -111,20 +111,18 @@ same as ``sha3(uint16(0x12))``.
 
 It might be that you run into Out-of-Gas for ``sha256``, ``ripemd160`` or ``ecrecover`` on a *private blockchain*. The reason for this is that those are implemented as so-called precompiled contracts and these contracts only really exist after they received the first message (although their contract code is hardcoded). Messages to non-existing contracts are more expensive and thus the execution runs into an Out-of-Gas error. A workaround for this problem is to first send e.g. 1 Wei to each of the contracts before you use them in your actual contracts. This is not an issue on the official or test net.
 
-.. index:: send, balance
+.. _address_related:
 
 Address Related
 ---------------
 
 ``<address>.balance`` (``uint256``):
-    balance of the address in Wei
+    balance of the :ref:`address` in Wei
 ``<address>.send(uint256 amount) returns (bool)``:
-    send given amount of Wei to address, returns ``false`` on failure
-
-For more information, see the section on :ref:`address`.
```

Comment:
But now it does not say anymore that more information can be found behind that link. Shouldn't we mention that?